src/ctia/store.clj

```diff
@@ -85,40 +85,23 @@
   (create-identity [this new-identity])
   (delete-identity [this org-id role]))
 
-;; core model
-(defonce judgement-store (atom nil))
-(defonce indicator-store (atom nil))
-(defonce feedback-store (atom nil))
-
-;; threats
-(defonce ttp-store (atom nil))
-(defonce campaign-store (atom nil))
-(defonce actor-store (atom nil))
-(defonce coa-store (atom nil))
-(defonce exploit-target-store (atom nil))
-
-;; sightings
-(defonce sighting-store (atom nil))
-
-;; incidents
-(defonce incident-store (atom nil))
-
-;; relations
-(defonce relation-store (atom nil))
-
-;; internal
-(defonce identity-store (atom nil))
-
-(def stores
-  {:judgement judgement-store
-   :indicator indicator-store
-   :feedback feedback-store
-   :ttp ttp-store
-   :campaign campaign-store
-   :actor actor-store
-   :coa coa-store
-   :exploit-target exploit-target-store
-   :sighting sighting-store
-   :incident incident-store
-   ;;:relation relation-store
-   :identity identity-store})
+(defonce stores (atom {:judgement []
+                       :indicator []
+                       :feedback []
+                       :ttp []
+                       :campaign []
+                       :actor []
+                       :coa []
+                       :exploit-target []
+                       :sighting []
+                       :incident []
+                       ;;:relation relation-store
+                       :identity []}))
+
+(defn write-store [store write-fn & args]
+  (first (doall (map #(apply write-fn % args) (store @stores)))))
+
+(defn read-store [store read-fn & args]
+  (apply read-fn (rand-nth (store @stores)) args))
```

Comment:
I really think this should be (first (store ...)).  We shouldn't be trying to do load balancing at this point.  The user should be able to deterministically specify the fastest store to be used in reads, and the easiest way to do that is to put it first in the list of stores.